subprojects/base-services/src/main/java/org/gradle/internal/progress/BuildOperationInternal.java

```diff
@@ -15,46 +15,15 @@
  */
 package org.gradle.internal.progress;
 
-import org.gradle.api.Nullable;
-
 /**
  *
  * This class is consumed by the build scan plugin.
  *
+ * TODO remove this subclass, consider moving complete content of this package to 'operations'
```

Comment:
I created an issue for the removal: https://github.com/gradle/gradle/issues/1859